releasedir/fs_blobs_dir.go

```diff
@@ -300,6 +304,23 @@ func (d FSBlobsDir) UploadBlobs() error {
 	return nil
 }
 
+func (d FSBlobsDir) containsSymlinks() bool {
+	files, err := d.fs.RecursiveGlob(filepath.Join(d.dirPath, "**/*"))
+	if err != nil {
+		return false
+	}
+
+	for _, file := range files {
+		fileInfo, _ := d.fs.Lstat(file)
```

Comment:
Seems like we're still ignoring the error here; is that intentional?

Given we know there's a file there, it seems like we should be able to expect a successful Lstat call. Anything else suggests an fs issue that we probably should be concerned with.